Sources/Clappr_iOS/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -557,27 +589,6 @@ extension AVFoundationPlayback {
         return ranges
     }
 
-    open override var supportDVR: Bool {
-        guard playbackType == .live else { return false }
-        return seekableTimeRanges.first(where: { $0.timeRangeValue.duration.seconds >= minDvrSize}) != nil
-    }
-
-    open override var dvrPosition: Double {
-        if let start = dvrWindowStart,
-            let end = dvrWindowEnd,
-            let position = player?.currentItem?.currentTime().seconds {
-            var calculatedPosition = (position - start) * 100
-            calculatedPosition = calculatedPosition / ((end - start) / 100)
-            calculatedPosition = (calculatedPosition * duration) / 10000
-            return calculatedPosition
-        }
-        return position
-    }
-
-    open override var currentDate: Date? {
-        return player?.currentItem?.currentDate()
-    }
-
     private var dvrWindowStart: Double? {
         return seekableTimeRanges.min { rangeA, rangeB in rangeA.timeRangeValue.start.seconds < rangeB.timeRangeValue.start.seconds }?.timeRangeValue.start.seconds
```

Comment:
I think it's risky to calculate the boundaries between seekableTimeRanges. The time ranges could be discontinuous, so the difference between the start of the first range and the end of the last range may not represent the duration.

Reply:
As I wrote in the previous reply, we will try to reproduce this scenario to test what values return and make sure that the logic works, otherwise find the right solution.